### DISCUSSION TITLE
Headless version

### DISCUSSION THREAD

**Discussion by dspangenberg** [inertiaui/modal#8] [General]
Do you plan a headless version with composables (e. g. like https://hybridly.dev/api/utils/use-dialog.html)? Even if your modal is largly customizable, more complex use cases like the responsive dialog from shadcn-vue (https://www.shadcnvue.com/docs/components/drawer.html) won't work.

**Comment by dspangenberg** [comment]
Oh, I've only just seen that you've already created a headless variant for React. I hope it will also be available for VueJS.

**Comment by pascalbaljet** [comment]
This is now available in both React and Vue: https://inertiaui.com/inertia-modal/docs/headless-mode
